### PR TITLE
Add a loader when REDEEMED POSITION PREVIEW is loaded

### LINKED ISSUE #742: Add a loader when REDEEMED POSITION PREVIEW is loaded
It may take some time to calculate/load a positions' amount to redeem on the Redeem position page
It would be nice to add a loader when a position to redeem is selected and amount is being loaded.

### PR DESCRIPTION
**Pull request by mariano-aguero**
Closes #742

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr743--conditionaltokensexplorer.review.gnosisdev.com

File: src/components/redeemPosition/PositionPreview/index.tsx [hunk 1]
```diff
@@ -18,9 +20,11 @@ interface Props {
   position: Maybe<PositionWithUserBalanceWithDecimals>
   condition: Maybe<GetCondition_condition>
   networkConfig: NetworkConfig
+  isLoading: boolean
 }
 
-export const PositionPreview = ({ condition, position }: Props) => {
+export const PositionPreview = (props: Props) => {
+  const { condition, isLoading, position } = props
   const { balanceERC1155, refetch } = useBalanceForPosition(position?.id || '')
```
Comment (hpmaxi) [inline src/components/redeemPosition/PositionPreview/index.tsx:28]: This could have a loading too?

**Review by hpmaxi — CHANGES_REQUESTED** [review]

**Comment by elena-zh** [comment]
The changes LGTM (from the UI side).

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr743--conditionaltokensexplorer.review.gnosisdev.com

**Comment by gabitoesmiapodo** [comment]
Fixed a little height / jumping issue with the spinner / preview container.

Otherwise, this looks good.

File: src/components/redeemPosition/PositionPreview/index.tsx (hunk 1 above)
Comment (mariano-aguero) [reply src/components/redeemPosition/PositionPreview/index.tsx]: Sure

**Review by hpmaxi — APPROVED** [review]

**Pull request merged**